requirements/requirements-neptune.txt 

```diff
@@ -0,0 +1 @@
+neptune-client
```

Comment:
could you please also add it to the workflow?
https://github.com/catalyst-team/catalyst/blob/master/.github/workflows/dl_cpu.yml#L98

Reply:
done.